Sparkle/SUCodeSigningVerifier.m

```diff
@@ -13,53 +13,62 @@
 
 @implementation SUCodeSigningVerifier
 
-+ (BOOL)codeSignatureMatchesHostAndIsValidAtPath:(NSString *)applicationPath error:(NSError *__autoreleasing *)error
++ (BOOL)codeSignatureAtPath:(NSString *)oldBundlePath matchesSignatureAtPath:(NSString *)newBundlePath error:(NSError *__autoreleasing *)error
 {
     OSStatus result;
     SecRequirementRef requirement = NULL;
     SecStaticCodeRef staticCode = NULL;
-    SecCodeRef hostCode = NULL;
+    SecStaticCodeRef oldCode = NULL;
     NSBundle *newBundle;
+    NSBundle *oldBundle;
     CFErrorRef cfError = NULL;
     if (error) {
         *error = nil;
     }
-
-    result = SecCodeCopySelf(kSecCSDefaultFlags, &hostCode);
-    if (result != noErr) {
-        SULog(@"Failed to copy host code %d", result);
+    
+    oldBundle = [NSBundle bundleWithPath:oldBundlePath];
+    if (!oldBundle) {
+        SULog(@"Failed to load NSBundle for original");
+        result = -1;
         goto finally;
     }
-
-    result = SecCodeCopyDesignatedRequirement(hostCode, kSecCSDefaultFlags, &requirement);
+    
+    result = SecStaticCodeCreateWithPath((__bridge CFURLRef)[oldBundle bundleURL], kSecCSDefaultFlags, &oldCode);
+    if (result == errSecCSUnsigned) {
+        return NO;
```

Comment:
Other branches set `result` and `goto finally`, so it'd be good to do it here for consistency